instruqt-tracks/terraform-cloud-aws-v2/16-clean-up-and-teardown/cleanup-workstation

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -l
```

Comment:
Should this cleanup go here or in `track_scripts`?

Reply:
This is legacy and should be removed from all tracks.